src/patternfly/components/DataList/data-list.scss

```diff
@@ -2,15 +2,25 @@ $pf-data-list--breakpoint--mobile: 380px;
 
 .pf-c-data-list {
   --pf-c-data-list--BackgroundColor: var(--pf-global--BackgroundColor--light-100);
+  --pf-c-data-list--BorderTopColor: var(--pf-global--BorderColor--light-200);
+  --pf-c-data-list--BorderTopWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-data-list--mobile--BorderTopWidth: var(--pf-global--spacer--sm);
+  --pf-c-data-list--BorderBottomColor: var(--pf-global--Color--light-200);
+  --pf-c-data-list--BorderBottomWidth: var(--pf-global--BorderWidth--sm);
+
+  @media screen and (max-width: $pf-global--breakpoint--mobile) {
+    --pf-c-data-list--BorderTopWidth: var(--pf-c-data-list--mobile--BorderTopWidth);
+  }
 
   // Item
   --pf-c-data-list__item--PaddingRight: var(--pf-global--spacer--xl);
-  --pf-c-data-list__item--mobile--PaddingRight: var(--pf-global--spacer--lg); // custom name for one-off breakpoint
+  --pf-c-data-list__item--mobile--PaddingRight: var(--pf-global--spacer--md);
   --pf-c-data-list__item--PaddingLeft: var(--pf-global--spacer--xl);
-  --pf-c-data-list__item--mobile--PaddingLeft: var(--pf-global--spacer--lg);
+  --pf-c-data-list__item--mobile--PaddingLeft: var(--pf-global--spacer--md);
   --pf-c-data-list__item--m-expanded--before--BackgroundColor: var(--pf-global--active-color--100);
   --pf-c-data-list__item-item--BorderTopColor: var(--pf-global--Color--light-200);
   --pf-c-data-list__item-item--BorderTopWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-data-list__item-item--mobile--BorderTopWidth: var(--pf-global--spacer--sm);
```

Comment:
I wonder if we should add a new `--pf-global--BorderWidth` var for this? Or is this just a one-off? If it's just a one-off, is this supposed to present itself as an 8px border, or more of an 8px gap that we would expect to change if someone decided to change `--pf-global--spacer--sm` here? I'm wondering if this should just be `#{pf-size-prem(8px)}` instead.

Reply:
I was thinking about this. Its a border and its used multiple times. I just added an xl border width variable. My only concern is that the variables increment by 1 each time:  sm - 1px, md - 2px, lg - 3px and now xl would be 8px which is a jump. How do you feel about making the xl - 4px and then I can calc and multiply it by 2. @mcoker

Reply:
after discussing this further we will be going with 8px hard coded height, because this is a one-off .. made those adjustments.